contrib/plasma-workspace/template.py

```diff
@@ -132,6 +136,41 @@ def post_install(self):
 
     self.rm(self.destdir / "usr/lib/systemd/user", recursive=True)
 
+    self.mkdir(
+        self.destdir / "usr/share/wallpapers/Chimera/contents/images",
+        parents=True,
+    )
+    self.mkdir(
+        self.destdir / "usr/share/wallpapers/Chimera/contents/images_dark"
+    )
+    self.install_file(
+        self.files_path / "metadata.json", "usr/share/wallpapers/Chimera"
+    )
+    self.install_link(
```

Comment:
what's the deal with the 2560x1600 number

Reply:
when i checked breeze's wallpapers, i noticed that for many of them (e.g. `/usr/share/wallpapers/Path/contents/images`), the various wallpapers sizes were all symlinks to `2560x1600.jpg` (and in only one case, `3200x2000.jpg`), so that's why I chose `2560x1600` as the size